### PR TITLE
News test

### PR DESCRIPTION
**Pull request by LehabuL**
I added tests for news app and also a back button to home page.

### CODE REVIEW

**Comment by BPamukcuoglu** [comment]
All tests are working, code is looking alright, waiting for the second review, thanks for your work 👍

**Review by BPamukcuoglu — APPROVED** [review]

**Review by yusuferdemnacar — APPROVED** [review]
I tested the changes and they are OK. Thanks.

**Pull request merged**
